src/redux/hooks/useEditingDistributionLimit.ts

```diff
@@ -70,5 +73,37 @@ export const useEditingDistributionLimit = (): [
     [defaultJBETHPaymentTerminal, dispatch],
   )
 
-  return [distributionLimit, setDistributionLimit]
+  const setDistributionLimitAmount = useCallback(
```

Comment:
Necessary addition to this hook to be able to set DL amount and currency separately. Using the current one was buggin